zhusuan/model/stochastic.py

```diff
@@ -571,3 +574,135 @@ def __init__(self,
             check_numerics=check_numerics,
         )
         super(Laplace, self).__init__(name, laplace, n_samples)
+
+
+class BinConcrete(StochasticTensor):
+    """
+    The class of univariate BinConcrete `StochasticTensor`.
```

Comment:
also  copy the descriptions in corresponding distributions (change links in it to  stochastictensors)